Crash to open entry form in android-current

![screenshot_20180422-094235](https://user-images.githubusercontent.com/5593590/39092597-c243f928-4611-11e8-8c14-428a0ca30a45.jpg)
Exception:
    java.lang.RuntimeException: android.os.TransactionTooLargeException: data parcel size 2567760 bytes
        at android.app.ActivityThread$StopInfo.run(ActivityThread.java:4203)
        at android.os.Handler.handleCallback(Handler.java:789)
        at android.os.Handler.dispatchMessage(Handler.java:98)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6809)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)
     Caused by: android.os.TransactionTooLargeException: data parcel size 2567760 bytes
        at android.os.BinderProxy.transactNative(Native Method)
        at android.os.BinderProxy.transact(Binder.java:787)
        at android.app.IActivityManager$Stub$Proxy.activityStopped(IActivityManager.java:4888)
        at android.app.ActivityThread$StopInfo.run(ActivityThread.java:4176)
        at android.os.Handler.handleCallback(Handler.java:789) 
        at android.os.Handler.dispatchMessage(Handler.java:98) 
        at android.os.Looper.loop(Looper.java:164) 
        at android.app.ActivityThread.main(ActivityThread.java:6809) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)

--- a/dhis2-android-app/src/main/java/org/dhis2/mobile/ui/activities/MenuActivity.java
+++ b/dhis2-android-app/src/main/java/org/dhis2/mobile/ui/activities/MenuActivity.java
@@ -137,6 +137,7 @@ public class MenuActivity extends BaseActivity implements OnNavigationItemSelect
         }
 
         super.onSaveInstanceState(outState);
+        outState.clear();
     }
 
     protected void attachFragment(Fragment fragment) {

